yarn-project/end-to-end/src/e2e_state_vars.test.ts

```diff
@@ -15,24 +21,47 @@ describe('e2e_state_vars', () => {
   beforeAll(async () => {
     ({ teardown, wallet } = await setup());
     contract = await DocsExampleContract.deploy(wallet).send().deployed();
-  }, 25_000);
+  }, 30_000);
 
   afterAll(() => teardown());
 
   describe('SharedImmutable', () => {
     it('private read of uninitialized SharedImmutable', async () => {
-      const s = await contract.methods.get_shared_immutable().view();
+      const s = await contract.methods.get_shared_immutable().simulate();
 
       // Send the transaction and wait for it to be mined (wait function throws if the tx is not mined)
       await contract.methods.match_shared_immutable(s.account, s.points).send().wait();
     });
 
-    it('private read of initialized SharedImmutable', async () => {
+    it('private read of SharedImmutable', async () => {
       await contract.methods.initialize_shared_immutable(1).send().wait();
-      const s = await contract.methods.get_shared_immutable().view();
 
```

Comment:
Could you add some comments to this test case? It's not really clear what you are testing here if you don't have the context.